rootfs/scheduler/resources/replicaset.py

```diff
@@ -4,7 +4,7 @@
 
 class ReplicaSet(Resource):
     api_prefix = 'apis'
-    api_version = 'extensions/v1beta1'
+    api_version = 'apps/v1'
```

Comment:
For backwards compatibility `api_version` should be moved into a propery method that checks if `self.version() >= parse("1.9.0")` is true to use `apps/v1`, similar to the code in the deployment resource.

Reply:
Yes, I will make that similar change soon. Maybe that code should live somewhere where it can be shared between Resources.

Reply:
I think the minor duplication is no big deal and not all resources depend on the same K8s versions for the api version, so sharing it is more likely to cause trouble.

Reply:
Good point as well. Maybe sharing across Resources may cause troubles.

Reply:
Done in d8e65a9